app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
           <li class= <%= 'active' if current_page?(students_path) %>><%= link_to "Students", students_path %></li>
           <li class= <%= 'active' if current_page?(user_invites_path) %>><%= link_to "Invites", user_invites_path %></li>
           <li><a href="#"></a></li>
-          <li><%= link_to "Log In", "/auth/github" unless current_user %></li>
+          <li><%= link_to "Log In", "/auth/github" unless current_user method: :post %></li>
           <li><%= link_to "Log Out", logout_path, method: :delete if current_user %></li>
```

Comment:
Does the `method: post` need to come before the `unless`?

Reply:
Yep!  That was a typo.